students/emorychristenson/lesson03/mailroom.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python3
+
+# Collection of existing donor data
+donors = [("John Jacob", [5.00, 1000.00, 4.25]),
+              ("Jingleheimer Schmidt", [240.50]),
+              ("Chanandler Bong", [15000.00, 4000.00]),
+              ("Jimni Christmas", [12.00]),
+              ("S. Claus", [7000.00, 14000.00, 200.00])]
+
+# Message that displays when script is run
+main_prompt = """
+Please choose from the below options.\n
+1 - Send a thank you
+2 - Create a report
+3 - Quit\n
+Type a number to select: """
+
+# Allows user to select existing donor, add a new donor, or list all donors from the collection
+def thank_you():
+    print("\nSend a thank you!\n")
+    response = input("""Please type the name of the donor you'd like to thank.\n
+Use 'list' to view current donors, or 'quit' to return to the menu: """)
+    while response.lower() == "list":
+        for name in donors:
+            print("\n",name[0])
+        response = input("\nPlease enter a donor name: ")
+    if response.lower() == "quit":
+        main()
+    else:
+        amount = float(input("\nEnter a donation amount: "))
+        
+    for donor in donors:
+        if response.lower() == donor[0].lower():
+            donor[1].append(float(amount))
+        else:        
+            donor = (response, [amount])
+            donors.append(donor)
+        send_email(donor, amount)
```

Comment:
This isn't adding to existing donor amounts but is creating new donors and donations